.circleci/run_e2e.sh

```diff
@@ -2,21 +2,38 @@
 
 CNI_HUB=${CNI_HUB:-istio}
 CNI_TAG=${CNI_TAG:-$1}
+CNI_ENABLE=${CNI_ENABLE:-true}
 
 # Dockerfile copies the PR's istio/cni helm chart into /go/helm/istio-cni
 chartdir=/go/helm
 
 cd /go/src/istio.io/istio
+if [[ "${ISTIO_REMOTE}" != "" ]]; then
+    git remote add nonorigin ${ISTIO_REMOTE}
+    git fetch nonorigin
+    git checkout nonorigin/${ISTIO_REMOTE_BRANCH:-master}
+fi
 
-# Install istio-cni prior to executing the Istio e2e test.  Now that the helm chart for istio/istio no longer
-# depends on the istio-cni chart, we need to explicitly do this as a prereq for installing Istio
-# (the e2e_simple test installs Istio).
-helm template --values ${chartdir}/istio-cni/values.yaml --name=istio-cni --namespace=kube-system --set "excludeNamespaces={}" --set hub=${CNI_HUB} --set tag=${CNI_TAG} --set pullPolicy=IfNotPresent --set logLevel=${CNI_LOGLVL:-debug}  ${chartdir}/istio-cni > istio-cni_install.yaml
-kubectl apply -f istio-cni_install.yaml
+echo "k8s version"
+kubectl version
+
+echo "k8s Nodes"
+kubectl get nodes
+
+if [[ "${CNI_ENABLE}" == "true" ]]; then
```

Comment:
There is now an option to have the test install the CNI for you.  You could possibly have used that but this is also fine.

Reply:
I think I need to do it this way because I need to use the exact CNI_HUB and CNI_TAG built and copied into the KDC nodes.

Reply:
You should be able to set the CNI_HUB and tag differently on the Istio side.    I recall that someone was arguing in my PR that you don't need different HUB and TAG between Istio and CNI and pretty sure I added the capability for just this type of thing.   It isn't a big deal if you don't want to change this,  but in theory you could let the istio side test infra do this for you.

Reply:
Forgot to mention that the other reason is to use the istio/cni helm charts from the changeset under test.